### PR TITLE
[14.0][MIG] account_menu_invoice_refund

### PR DESCRIPTION
**Pull request by sergiocorato**

### CODE REVIEW

**Comment by sergiocorato** [comment]
replace #950  as stale

**Review by GianmariaBucceri — APPROVED** [review]

File: account_menu_invoice_refund/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
```
Comment (SkiBY) [inline account_menu_invoice_refund/__init__.py:1]: Why do we need this file in i18n?

Comment (sergiocorato) [reply account_menu_invoice_refund/__init__.py:1]: This file is in root folder of the module, not in i18n, and is needed by default for python logic.

Comment (SkiBY) [reply account_menu_invoice_refund/__init__.py:1]: O! I'm so sorry, lost my sight

**Review by SkiBY — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests

**Comment by AaronHForgeFlow** [comment]
@sergiocorato can you fix the commit message of the migration? it should be something like "[MIG] $MODULE: Migration to 14.0"

**Comment by sergiocorato** [comment]
> @sergiocorato can you fix the commit message of the migration? it should be something like "[MIG] $MODULE: Migration to 14.0"

Done! Thanks.

**Comment by AaronHForgeFlow** [comment]
@sergiocorato thank you!

/ocabot migration account_menu_invoice_refund
/ocabot merge nobump

**Pull request merged**